target/linux/ramips/dts/RB750Gr3.dts

```diff
@@ -67,63 +68,95 @@
 		};
 
 		usb {
-			gpio-export,name = "usb";
+			gpio-export,name = "usb_power_off";
 			gpio-export,output = <1>;
-			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+		};
+
+		// Used to enable power-over-ethernet passthrough to eth4 (according to OEM source).
+		poe_out {
+			gpio-export,name = "poe_passthrough";
+			gpio-export,output = <0>;
+			gpios = <&gpio0 17 GPIO_ACTIVE_HIGH>;
+		};
+
+		poe_status {
```

Comment:
The whole gpio-export is a hack that was rejected upstream a long time ago.

As long as we don't need to set GPIOs to a specific value in the early (kernel) boot process (like usb), I would prefer to not add the POE status.

We will need to find something else for such use cases. It might be a good idea to document the GPIO in the commit message.

Reply:
I guess, I will just omit the poe thing in this patch then